### DISCUSSION TITLE
How the buble free energy can be calculated?

### DISCUSSION THREAD

**Discussion by gangliu-github** [ttadano/alamode#197] [Q&A]
In the website, I see the BUBBLE-tag can be used to calculate bubble correction on top of the SCPH dynamical matrix. But I found the free energy keep unchanged with BUBBLE=1, while the phonon bands become different. Then, can the buble free energy be calculated with alamode 1.5.0?
By the way, are the 3rd and 4th FC both nesessary for the calculation of bubble correction?

**Comment by AinDin64** [comment]
Hello, may I ask if your problem has been solved? I have the same issue. It seems that version 1.5 doesn't support it, but version 1.3 does.

**Reply by gangliu-github** [reply]
> Hello, may I ask if your problem has been solved? I have the same issue. It seems that version 1.5 doesn't support it, but version 1.3 does.

I haven‘t solved the problem till now.
